### ISSUE TITLE
XGBoost - Biniam

### ISSUE THREAD

**Issue by Micky373** [Micky373/abtest-mlops#7]
Split the data into 70% training, 20% validation, and 10% test sets. 

- Based on the reading material provided, apply machine learning to the training data. Train a machine learning model using 5-fold cross-validation using XGBoost.

- Define the appropriate loss function for the model using the validation data. 

- Compute feature importance - what’s driving the model? Which parameters are important predictors for the different ML models? What contributes to the goal of gaining more “Yes” results?

- Which data features are relevant to predicting the target variable?
- For each of the ML algorithms above, find the best model by tuning their hyperparameters and each time adding the tried models in MLFlow.
